### ISSUE TITLE
[aesthetic] Removal of the top white bar in Windows

### ISSUE THREAD

**Issue by CyberSecGuy** [mccxiv/tc#518]
I love this, it's amazing! Hiding the top white windows bar would be amazing especially for night-mode users.

https://i.imgur.com/MhtYUCO.png

**Comment by Bauke** [comment]
Just gonna leave [this tutorial here], it explains from start to finish how you'd implement a custom title bar in Electron, then we could style it according to the Tc theme.

[this tutorial here]: https://github.com/binaryfunt/electron-seamless-titlebar-tutorial

**Comment by mccxiv** [comment]
Hmm yes. Rather than removing, I guess what we want is a custom implementation. I'm always weary of reimplementing native things, but many (most?) popular electron apps seem to do it, like Slack & Discord

**Comment by Bauke** [comment]
Yeah, I don't think removing the title bar is a smart thing. I think having a setting like Visual Studio Code that lets you choose between native or custom would probably be the best.
